core/src/test/java/io/questdb/griffin/AsyncOffloadTest.java

```diff
@@ -122,6 +119,14 @@ public static void setUpStatic() {
         AbstractGriffinTest.setUpStatic();
     }
 
+    @Before
+    public void setUp() {
```

Comment:
Even though they're set once in setUpStatic they get reverted back to defaults in tearDown() so this method is needed, otherwise running single or several methods test   yields different result to running whole test class.

Reply:
I was under an impression that we only read them once, but indeed some of the classes like `DataFrameRecordCursorFactory` read them when being constructed. Thanks for the clarification.